app/controllers/application_controller.rb

```diff
@@ -14,7 +14,7 @@ class ApplicationController < ActionController::Base
 
     def set_locale
       locale = locale_from_accept_language
-      I18n.locale = (I18n.available_locales.include? locale.to_sym) ? locale.to_sym : I18n.default_locale
+      I18n.locale = (I18n.available_locales.include?(locale.to_sym) ? locale.to_sym : I18n.default_locale)
```

Comment:
__[reek]__
* [FeatureEnvy] refers to 'locale' more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)

<!-- Comment by [SideCI](https://sideci.com) -->